state/block_test.go

```diff
@@ -66,3 +68,118 @@ func TestGetBlockByNumber(t *testing.T) {
 	}
 
 }
+
+func TestAddBlock(t *testing.T) {
+	dataDir, err := ioutil.TempDir("", "TestAddBlock")
+	if err != nil {
+		t.Fatal("Failed to create temp folder for TestAddBlock test", "err", err)
+	}
+
+	//Create a new blockState
+	blockState, err := NewBlockState(dataDir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	//Close DB
+	defer func() {
+		err = blockState.db.Db.Close()
+		if err != nil {
+			t.Fatal("BlockDB close err: ", err)
+		}
+	}()
+
+	// Create header
+	header0 := &types.BlockHeader{
+		Number: big.NewInt(0),
+	}
+	// Create blockHash
+	blockHash0 := header0.Hash()
+
+	// BlockBody with fake extrinsics
+	blockBody0 := types.BlockBody{}
+	blockBody0 = append(blockBody0, []byte{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}...)
```

Comment:
this can probably be simplified into one line: `blockBody0 := types.BlockBody{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}`

Reply:
yep, good point